### DISCUSSION TITLE
增加对S390X架构的CPU支持

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#1718] [Ideas]
RT

Go的编译可以编译s390x的二进制文件，希望v2fly项目组官方跟进！

**Comment by AkinoKaede** [comment]
In fact, V2Ray used to provide binary executables compiled for the S390x platform, but this was removed after the release of v4.34.0.

[v4.34.0 Release Note](https://github.com/v2fly/v2ray-core/releases/tag/v4.34.0)
[compiling guide](https://www.v2fly.org/developer/intro/compile.html)
